src/python/pants/backend/jvm/targets/jvm_binary.py

```diff
@@ -335,6 +336,8 @@ def __init__(self,
       (aka monolithic aka fat) binary jar. The order of the rules matters: the first rule which
       matches a fully-qualified class name is used to shade it. See shading_relocate(),
       shading_exclude(), shading_relocate_package(), and shading_exclude_package().
+    :param list extra_jvm_options: A list of key value pairs of jvm options to use when running the
```

Comment:
Probably clearer not to refer to "key-value pairs"... sometimes they will be key-value pairs, but saying that makes it sound like it's not a collection of "literal options to be passed to the JVM". The example is good, but it might also be good to include an `-Xmx4g` option, to make it clear that any JVM option is supported.

Reply:
I agree with adding the plain flag case to the test project, but for the comment wording I was following the example set here: https://github.com/blorente/pants/blob/e5a5e5ed40ae7312d523207c4ddd888909c6ff8c/src/python/pants/backend/jvm/targets/junit_tests.py#L54

I agree that the nomenclature is misleading, I'll change both places.